containers/python/rpc.py

```diff
@@ -511,10 +514,19 @@ def start(self, model, host, port, model_name, model_version, input_type):
         self.server.model_input_type = model_input_type
         self.server.model = model
 
+        # Create a file named model_is_ready.check to show that model and container
+        # are ready
+        # os.system("touch model_is_ready.check")
+        with open("/model_is_ready.check", "w") as f:
+            f.write("READY")
         child_conn, parent_conn = Pipe(duplex=False)
         metrics_proc = Process(target=run_metric, args=(child_conn, ))
         metrics_proc.start()
-        self.server.run(parent_conn)
+        try:
+            self.server.run(parent_conn)
+        except Exception as e:
+            os.system("rm model_is_ready.check")
```

Comment:
Use a Python method and also the path is wrong. Change to:
```py
os.remove("/model_is_ready.check")
```

Reply:
Done